packages/eslint-plugin/tests/rules/directive-class-suffix.test.ts

```diff
@@ -24,48 +24,50 @@ ruleTester.run(RULE_NAME, rule, {
       selector: 'sgBarFoo'
     })
     class TestDirective {}
-`,
+    `,
     `
     @Directive({
       selector: 'sgBarFoo'
     })
     class TestValidator implements Validator {}
-`,
+    `,
     `
     @Directive({
       selector: 'sgBarFoo'
     })
     class TestValidator implements AsyncValidator {}
-`,
+    `,
     `
-    @Directive
-    class TestDirective {}
-`,
+    @Directive()
```

Comment:
You seem to have removed the `@Directive` case, please add it back I think it's worth having just so we know how the linter behaves.

Reply:
Please also add a comment next to the new `@Directive()` case and reference back to the issue discussion

Reply:
> You seem to have removed the @Directive case, please add it back I think it's worth having just so we know how the linter behaves.

Hmm, yes I've removed because it doesn't even compile as you can see here: 

![Captura de Tela 2021-04-06 às 19 54 21](https://user-images.githubusercontent.com/11965907/113787711-e65c9f00-9711-11eb-9abc-09e71502000e.png)

[**Repro link**](https://stackblitz.com/edit/angular-ivy-oipuw1)

But, let me know if you want me to add it back anyway.

> Please also add a comment next to the new `@Directive()` case and reference back to the issue discussion

Added the link, let me know if we need an additional description as well.

Reply:
The Angular tooling doesn't like it but it is valid TypeScript, so it's still good to understand how our tooling behaves when it exists, right?

Reply:
No problem. Added it back.